src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Unix.cs

```diff
@@ -23,6 +23,18 @@
         private const string TimeZoneEnvironmentVariable = "TZ";
         private const string TimeZoneDirectoryEnvironmentVariable = "TZDIR";
         private const string FallbackCultureName = "en-US";
+        private const string GmtId = "GMT";
+        private static readonly Lazy<CultureInfo> s_LazyFallbackCulture = new Lazy<CultureInfo>(() => new CultureInfo(FallbackCultureName));
```

Comment:
>s_LazyFallbackCulture [](start = 50, length = 21)

we don't need to do this lazy initialization at all. just define s_FallbackCulture = CultureInfo.GetCultureInfo(FallbackCultureName);